README.md

```diff
@@ -257,9 +257,9 @@ redisClusterReset(clusterContext);
 ## Cluster asynchronous API
 
 Hiredis-cluster comes with an asynchronous cluster API that works with many event systems.
-Currently there are adapters that enables support for `libevent`, `libev`, `libuv` and
-Redis Event Library (`ae`), but more can be added. The hiredis library has adapters for
-additional event systems that easily can be adapted for hiredis-cluster as well.
+Currently there are adapters that enables support for `libevent`, `libev`, `libuv`, `glib2`
+and Redis Event Library (`ae`), but more can be added. The hiredis library has adapters
```

Comment:
There are no examples, so I suggest we link to the tests somewhere here.

```suggestion
and Redis Event Library (`ae`). For usage examples, see the test programs with the
different event libraries `tests/ct_async_{libev,libuv,glib2}.c`.

The hiredis library has adapters
```

Reply:
Ah, good.
We are currently only requiring libevent to be able to test the async api.
The other libs are optional, but this info can be added to the README

Reply:
Right... so I guess the prereqs for tests are OK then. The docs could refer to the tests as examples though.

Does make test print "Skipped" for the tests for the adapters which are missing?

Reply:
Thanks, I added the references to the tests in the README and found a way now in cmake/ctest to visualize that the adapter tests are disabled/skipped.
Example when libev is missing:
```
% make test
Running tests...
Test project /home/bjorn/git/hiredis-cluster/build
      Start  1: ct_async
 1/15 Test  #1: ct_async .........................   Passed    0.00 sec
      Start  2: ct_commands
...
 7/15 Test  #7: ct_async_libuv ...................   Passed    0.01 sec
      Start  8: ct_async_libev
 8/15 Test  #8: ct_async_libev ...................***Not Run (Disabled)   0.00 sec
      Start  9: ct_async_glib
 9/15 Test  #9: ct_async_glib ....................   Passed    0.01 sec
      Start 10: set-get-test
...
      Start 15: dbsize-to-all-nodes-test
15/15 Test #15: dbsize-to-all-nodes-test .........   Passed    0.02 sec

100% tests passed, 0 tests failed out of 14

Label Time Summary:
CT    =   0.43 sec*proc (8 tests)

Total Test time (real) =   0.58 sec

The following tests did not run:
	  8 - ct_async_libev (Disabled)
```

Reply:
Looks good! So, before, it was just excluded and it would appear that all tests passed and no tests were skipped?

Reply:
> Looks good! So, before, it was just excluded and it would appear that all tests passed and no tests were skipped?

Previously the test binaries were not built and not added as tests to CMakes test runner, so not visible in the list.
The list was just shorter..
By giving CMake a fake name (to the non-build binary) when adding the disabled test its at least visible.